lib/puppet/type/cs_primitive.rb

```diff
@@ -166,10 +166,7 @@
   end
 
   autorequire(:cs_shadow) do
-    autos = []
-    autos << @parameters[:cib].value if @parameters[:cib]
-
-    autos
+    [@parameters[:cib]]
```

Comment:
Is this something we could potentially do for https://github.com/voxpupuli/puppet-filemapper/pull/17/? There's a similar line using super.